starters/express-typescript-typeorm-postgres/src/controllers/technology/handlers/delete-technology.spec.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
No, this is a test file and the mocks need the any type.

Reply:
It is bad practice in files that go to production, but please see the whole file.
These are unit tests, and I am creating mock `Request` and `Response` objects and a whole mock data repository with only one or two functions on them. I want to test the functions, not spending time on creating a mock object that has the same type as an original express Request object would have.

Reply:
@ihardz check out the branch, add a type and see that then typescript will scream when you put that variable into the function, and then you need to have `getTechnologies(MOCK_REQUEST as any, MOCK_RESPONSE as any, MOCK_NEXT_FN as any)`

If you create types, you _must_ create types that are the same as the types from express. Spending time on creating mocks that mock a large type is useless. `any` is exactly for these kinds of situations.

Reply:
<img width="1098" alt="Screenshot 2022-12-02 at 13 39 16" src="https://user-images.githubusercontent.com/13385210/205295137-28479c87-23d7-4ee1-80a8-7918d56c3034.png">

Reply:
Hi both @ihardz @TapaiBalazs ! Thank you both for the above discussion as it helps drive better software!!! :)

I am going to take ownership of this and provide my decision going forward...(i am also going to add this in the chat so that we can chat more if needed, but for this ticket please go ahead with what I am about to define).

1) If the type that needs to be mocked is a 3rd party library (like in this case express), It is ok to use ANY as mocking it would take time, be verbose and add very improvement for the time spent.
2) If the type is actually an internal type (one from our app), we should try to actually use the correct type and just mock it in the special occasion when we know that the investment is not worth the cause (like a large 20 parameter type from which we need just one). In this case the any should just be defined at a mock level and not for the whole page!

Thank you again for the above discussion..